netbout-web/src/main/java/com/netbout/rest/Markdown.java

```diff
@@ -90,7 +90,7 @@ public String html() {
                 new PegDownProcessor().markdownToHtml(
                     Markdown.formatLinks(this.text)
                 )
-            );
+            ).replaceAll("(</a>)(\\w)", "$1 $2");
```

Comment:
@antonini isn't it better to compile the regex and leave it as a static field? This way it will re-compile and match the regex always...